Modules/_ctypes/stgdict.c

```diff
@@ -337,6 +337,26 @@ MakeAnonFields(PyObject *type)
     return 0;
 }
 
+/*
+  Append {padding}x to the PEP3118 format string.
```

Comment:
Please could this description be expanded to note the behaviour on failure, and to note that the returned pointer refers to newly-allocated memory (that it's then the caller's responsibility to deal with)?

Reply:
In the style of `_ctypes_alloc_format_string`:
```suggestion
  Allocate a memory block for a pep3118 format string, copy prefix (if
  non-null) into it and append `{padding}x` to the end. 
  Returns NULL on failure, with the error indicator set.
```

Reply:
Looks good; thank you.